ras/ras_ppcutils.py

```diff
@@ -291,10 +291,22 @@ def test_ls_vdev(self):
         """
         self.log.info("===============Executing ls-vdev tool test===="
                       "===========")
+        self.is_fail = 0
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
-        self.error_check()
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
+        flag = True
+        for line in self.run_cmd_out("lsblk").splitlines():
+            if dev_name in line:
+                if dev_name == line.split()[0]:
```

Comment:
checking once is enough I believe , ones you check device is in line and than again eqaute to device name .. both looks same

Reply:
if dev_name == line.split()[0]
The above one takes the first word(device name) and checks for the exact match of the device name like sda = sda 
if I don't include this condition there might be a chances of matching sda1 or sda2..which are not exact matches